bin/rebuild-package-locks.js

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
```

Comment:
@raymondfeng , why do we need to rebuild package locks?
Is this a script we should make available to anyone building an LB4 app?

Reply:
The shopping project is set up as a lerna monorepo and it requires a few steps to rebuild package-lock files.